### PR TITLE
Updated (storage capacity ga)

### PR DESCRIPTION
**Pull request by Krishnamohan-Yerrabilli**
### Why:

miswritten, unwanted articles usage

### What's being changed

Changed " about remaining capacity" to " about the remaining capacity" -> article usage fixed

Changed "slowed down scale" to "slowed downscale" -> miswritten replaced to correct form.

Added " For this a better" to "For this, a better" -> Introductory phase adding comma would be appropriate

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the self-review checklist.

### CODE REVIEW

File: content/en/blog/_posts/2022-05-06-storage-capacity-GA/index.md [hunk 1]
```diff
@@ -56,12 +56,12 @@ Also not solved is support in Cluster Autoscaler for Pods with volumes. For CSI
 drivers with storage capacity tracking, a prototype was developed and discussed
 in [a PR](https://github.com/kubernetes/autoscaler/pull/3887). It was meant to
 work with arbitrary CSI drivers, but that flexibility made it hard to configure
-and slowed down scale up operations: because autoscaler was unable to simulate
+and slowed downscale up operations: because autoscaler was unable to simulate
```
Comment (sftim) [inline content/en/blog/_posts/2022-05-06-storage-capacity-GA/index.md:59]: The original text is idiomatic; this revision doesn't make sense.

File: content/en/blog/_posts/2022-05-06-storage-capacity-GA/index.md [hunk 2]
```diff
@@ -56,12 +56,12 @@ Also not solved is support in Cluster Autoscaler for Pods with volumes. For CSI
...
 volume provisioning, it only scaled the cluster by one node at a time, which
 was seen as insufficient.
 
 Therefore that PR was not merged and a different approach with tighter coupling
-between autoscaler and CSI driver will be needed. For this a better
+between autoscaler and CSI driver will be needed. For this, a better
```
Comment (sftim) [inline content/en/blog/_posts/2022-05-06-storage-capacity-GA/index.md:64]: I think this addition is OK, except that adding just a comma to a blog article is a [trivial change](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits) that wouldn't be worth making.

**Review by sftim — COMMENTED** [review]
For blog articles, authors write in their own voice. So long as the grammar and meaning is clear, we're OK with wording this isn't perfect English.

I'm afraid that we shouldn't make these changes and don't need to amend this article.

**Comment by Krishnamohan-Yerrabilli** [comment]
I'm closing this PR, I'm able to improve myself from with this kind of suggestions 
 Thank you for your valuable feedback @sftim

File: content/en/blog/_posts/2022-05-06-storage-capacity-GA/index.md (hunk 2 above)
Comment (Krishnamohan-Yerrabilli) [reply content/en/blog/_posts/2022-05-06-storage-capacity-GA/index.md:64]: Thank you for your valuable feedback @sftim

**Pull request closed without merging**